src/model/formats/test.ui.json

```diff
@@ -0,0 +1,337 @@
+{
+  "metadata": {
+    "name": "Test checklist name",
```

Comment:
Also very minor: mention somewhere in the metadata that this is meant for manual testing?